### PR TITLE
get tests working again mostly

### PR DESCRIPTION
**Pull request by zsimjee**

### CODE REVIEW

File: .github/workflows/pr_qc.yml [hunk 1]
```diff
@@ -38,3 +37,4 @@ jobs:
           if [ -z "$diff" ]; then echo "Missing release notes! exiting..."; exit 1; fi
         env:
           NODE_AUTH_TOKEN: ${{ secrets.TS_GH_BOT }}
+          NODE_OPTIONS: "--max-old-space-size=8192"
```
Comment (CalebCourier) [inline .github/workflows/pr_qc.yml:40]: Because of the virtual dom?

File: .github/workflows/pr_qc.yml [hunk 2]
```diff
@@ -24,7 +24,6 @@ jobs:
       - run: |
           npm ci;
           npm run qa;
-          npm run build;
```
Comment (CalebCourier) [inline .github/workflows/pr_qc.yml:27]: We should add this back.  The point of running it as part of QC is to verify that the changes in the PR can/do build.

Comment (zsimjee) [reply .github/workflows/pr_qc.yml:27]: Yes! It got left undone after I bumped the VM size. Adding it back!

File: .github/workflows/pr_qc.yml (hunk 1 above)
Comment (zsimjee) [reply .github/workflows/pr_qc.yml:40]: Yes! Isn’t it shocking that it needs this much room??

File: package.json [hunk 3]
```diff
@@ -76,6 +75,8 @@
     "@aws-sdk/client-sts": "^3.319.0",
     "@aws-sdk/types": "^3.257.0",
     "@chakra-ui/icons": "^2.0.17",
+    "@chakra-ui/react": "^2.6.1",
+    "@tinystacks/ops-core-widgets": "0.0.11",
```
Comment (CalebCourier) [inline package.json:79]: Do we need specific versions of these packages? Or can we keep these as peer dependencies?

Comment (nefertitirogers) [reply package.json:79]: I think we can keep them as peer dependencies, I was getting errors locally for them not being installed but I think it's my problem. So let me update this

**Comment by CalebCourier** [comment]
This still isn't building correctly.  I copied over what we are doing in https://github.com/tinystacks/ops-core-widgets locally and the build output was correct but then many of the tests either failed or didn't run.

**Comment by zsimjee** [comment]
Reviewed, this looks good to go! Merging up, we'll go ahead and merge other PRs down into main once that's done

**Pull request merged**